src/Microsoft.OData.Edm/PropertyAliasHelpers.cs

```diff
@@ -0,0 +1,42 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="PropertyAliasHelpers.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Linq;
+
+namespace Microsoft.OData.Edm
+{
+    internal static class PropertyAliasHelpers
+    {
+        internal static IEdmStructuralProperty GetKeyProperty(IEdmStructuralProperty property, string[] keyPropertySegments, string propertyAlias)

```

Comment:
Document this method so it's clear what it does

Reply:
Is this method called from multiple places? If no, would it have been better for it to reside in the class from where it's called seeing it's the only method in this class and file...